xbmc/utils/IBacklight.h

```diff
@@ -0,0 +1,16 @@
+#include "ServiceBroker.h"
+
+#include <memory>
+
+class IBacklight
+{
+public:
+  static void Register(std::shared_ptr<IBacklight> backlight)
+  {
+    CServiceBroker::RegisterBacklight(backlight);
+  }
+
+  virtual bool SetBrightness(int brightness) = 0;
+  virtual int GetBrightness() const = 0;
+  virtual int GetMaxBrightness() const = 0;
```

Comment:
`int` is not very descriptive of what a brightness is, so could use some documentation. typing could also improve semantics. could I suggest a float in the range [0, 1]?